scipy/linalg.pxd

```diff
@@ -1 +1 @@
-from .linalg cimport cython_blas, cython_lapack
+from scipy.linalg cimport cython_blas, cython_lapack
```

Comment:
The stated preference in #17234 was to keep using relative imports, so that this can never accidentally pick up some other installed scipy.